docs/cli_interface.rst

```diff
@@ -10,8 +10,8 @@ CLI flags
 
 It modifies the environment variables in a shell to create an isolated Python environment, so you'll need to have a
 shell to run it. You can type in ``virtualenv`` (name of the application) followed by flags that control its
-behaviour. All options have sensible defaults, so you'll get a working virtual environment in a ``venv`` folder even
-if you don't pass any options. The default values for the command line options can be overridden via the
+behaviour. All options have sensible defaults, and there's on required argument: then name/path of the virtual
```

Comment:
```suggestion
behaviour. All options have sensible defaults, and there's one required argument: then name/path of the virtual
```

Reply:
Also, "the name/path", not "then name/path"